master/internal/db/postgres_users.go

```diff
@@ -349,7 +349,7 @@ func (db *PgDB) UserByID(userID model.UserID) (*model.FullUser, error) {
 	var fu model.FullUser
 	if err := db.query(`
 SELECT
-	u.id, u.username, u.admin, u.active,
+	u.id, u.username, u.display_name, u.admin, u.active,
```

Comment:
👍 good catch. fwiw, this sort of bug was exactly one of the reasons to move to bun.